task.js

```diff
@@ -0,0 +1,143 @@
+const {promisify} = require('util');
```

Comment:
I don't think these functions and this file is needed, because majority of code is the same for both JS and TS templates. You could keep the old functions and accept `typescript` option to use different file extension or install different dependencies.

For example:


```js
const install = async ({typescript}) => {
	const jsDependencies = [...];
	const tsDependencies = [...];

	await execa('npm', ['install', ...(typescript ? tsDependencies : jsDependencies)]);
};
```

Reply:
Ah that's fair, I did find the verboseness rather nice for maintaining both template choices (a la Listr enabled flag). Typescript does have one additional install step as well. However, will look at how this can be reformatted to be more dry.